config/routes.rb

```diff
@@ -72,6 +72,7 @@
   resource :admin, :except => [:show] do
     match "email_instructors", via: [:get, :post]
     match "github_integration", via: [:get]
+    get "clear_cache"
```

Comment:
This should be a `post` request to prevent CSRF attacks

Reply:
Fixed.